sirepo/resource.py

```diff
@@ -44,6 +45,27 @@ def glob_paths(*paths):
     )
 
 
+def render(*paths, target_dir=None, j2_ctx=None):
+    """Render a jinja resource into target_dir
+
+    Args:
+        paths (str): Path components of resource file without pykern.pkjinja.RESOURCE_SUFFIX
+        target_dir (py.path): target directory for rendered file
+        j2_ctx (PKDict): parameters to jinja file
+
+    Returns:
+        py.path: path to rendered file
+    """
+    f = paths[-1]
+    t = target_dir.join(f)
```

Comment:
Change `t`  to `res`, since we generally use that as a convention for the return value.